pom.xml

```diff
@@ -58,24 +58,25 @@
       <url>https://oss.sonatype.org/content/repositories/snapshots</url>
     </snapshotRepository>
     <site>
-      <id>cask</id>
-      <url>http://cask.co</url>
+      <id>CDAP</id>
+      <url>http://cdap.io</url>
     </site>
   </distributionManagement>
 
   <issueManagement>
-    <url>https://issues.cask.co/browse/HYDRATOR</url>
+    <url>https://issues.cask.co</url>
   </issueManagement>
   <properties>
     <jrecord.version>0.80.8</jrecord.version>
     <cb2xml.version>0.80.8-jrecord</cb2xml.version>
     <main.basedir>${project.basedir}</main.basedir>
-    <cdap.version>4.1.0</cdap.version>
+    <cdap.version>6.0.0-SNAPSHOT</cdap.version>
+    <guava.version>19.0</guava.version>
     <hadoop.version>2.3.0</hadoop.version>
-    <hydrator.version>1.7.0</hydrator.version>
+    <hydrator.version>2.2.0-SNAPSHOT</hydrator.version>
     <widgets.dir>widgets</widgets.dir>
     <docs.dir>docs</docs.dir>
-    <app.parents>system:cdap-etl-batch[4.1.0-SNAPSHOT,5.0.0-SNAPSHOT),system:cdap-etl-realtime[4.1.0-SNAPSHOT,5.0.0-SNAPSHOT),system:cdap-data-pipeline[4.1.0-SNAPSHOT,5.0.0-SNAPSHOT),system:cdap-data-streams[4.1.0-SNAPSHOT,5.0.0-SNAPSHOT)</app.parents>
+    <app.parents>system:cdap-data-pipeline[6.0.0-SNAPSHOT,7.0.0-SNAPSHOT),system:cdap-data-streams[6.0.0-SNAPSHOT,7.0.0-SNAPSHOT)</app.parents>
```

Comment:
This is not used I think. The same goes for widgets.dir and docs.dir.
I think these are some legacy properties, which care not needed now.

Reply:
this property used maven-antrun-plugin in javascript

Reply:
we need to remove javascript usage. Please read the comment below.